src/client/src/apps/MainRoute/widgets/spotTile/components/analyticsTile/AnalyticsTilePriceControl.tsx

```diff
@@ -32,31 +37,54 @@ const AnalyticsPriceControl: React.SFC<Props> = ({
     currencyPair.pipsPosition,
     currencyPair.ratePrecision,
   )
+  const {
+    isRfqStateReceived,
+    isRfqStateExpired,
+    isRfqStateCanRequest,
+    isRfqStateNone,
+  } = getConstsFromRfqState(rfqState)
+
   return (
-    <AnalyticsPriceControlHeader>
+    <AnalyticsPriceControlHeader data-qa="analytics-tile-price-control__header">
       <PriceMovement
         priceMovementType={priceData.priceMovementType}
         spread={spread.formattedValue}
       />
       <div>
-        <PriceButton
-          handleClick={() => executeTrade(Direction.Sell, priceData.bid)}
-          direction={Direction.Sell}
-          big={bidRate.bigFigure}
-          pip={bidRate.pips}
-          tenth={bidRate.pipFraction}
-          rawRate={bidRate.rawRate}
-          disabled={disabled}
-        />
-        <PriceButton
-          handleClick={() => executeTrade(Direction.Buy, priceData.ask)}
-          direction={Direction.Buy}
-          big={askRate.bigFigure}
-          pip={askRate.pips}
-          tenth={askRate.pipFraction}
-          rawRate={askRate.rawRate}
-          disabled={disabled}
-        />
+        {isRfqStateCanRequest && (
+          <PriceButtonDisabledPlaceholder data-qa="price-controls__price-button-disabled-1">
+            <Icon className="fas fa-ban fa-flip-horizontal" />
+            Streaming price unavailable
```

Comment:
=> `Price streaming unavailable`?

Reply:
I kept it consistent with the other workspace view
![Screen Shot 2019-09-23 at 10 18 45 AM](https://user-images.githubusercontent.com/38663839/65433759-9b0ec400-ddeb-11e9-8f52-4f438ffd2117.png)
![Screen Shot 2019-09-23 at 10 19 33 AM](https://user-images.githubusercontent.com/38663839/65433780-a8c44980-ddeb-11e9-80a3-2cba30212f8f.png)

Reply:
Fair enough :+1